### DISCUSSION TITLE
Is there a way to access the checksums of artifacts?

### DISCUSSION THREAD

**Discussion by crschnick** [jreleaser/jreleaser#1376] [Q&A]
I wanted to implement a multi-arch homebrew build like this: https://github.com/Homebrew/homebrew-cask/blob/f1d51e61c828dfb61690e94a72984b0ee964d907/Casks/1password.rb

I can do everything with a custom template however I would have to manually calculate and pass the checksums as there's more than one artifact now. Is there way to access the checksums of jreleaser in gradle or mustache?

**Comment by aalmiray** [comment]
Yes, there is. You can refer to an artifact by using the following patterns https://jreleaser.org/guide/latest/reference/name-templates.html#_artifact

Specifically, you're looking for those that match `artifact${Platform}Checksum${SHA}`.
